### PR TITLE
User profile is completed

### PR DESCRIPTION
**Pull request by shubhamivane**
user profile and upcoming contest CLI feature is implemented and a module is made to make all API calls.

### CODE REVIEW

File: cf/api/user.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import api
+
+
+def user_profile(user_handle):
+    """ Printing user profile from Codeforces """
+    end_point_url = 'https://codeforces.com/api/user.info?handles={}'.format(user_handle)
+    json_response, error = api.call_api(end_point_url)
+    if json_response is not None:
```
Comment (kakshay21) [inline cf/api/user.py]: `if not json_response:`

File: cf/api/api.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import requests
+import json
+
+def call_api(api_end_point):
+    """ Request given API endpoint and return json result """
+    https_errors_codes = {
+        400: 'Bad Request : The server did not understand the request.',
```
Comment (kakshay21) [inline cf/api/api.py]: Try using exception provided by the `request` module
https://stackoverflow.com/questions/16511337/correct-way-to-try-except-using-python-requests-module

File: cf/api/contest.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            contest_list = list(map(lambda contest: (contest[1], convert_sec_to_date(contest[0])), contest_list)) 
+        else:
+            contest_list[0][0], contest_list[0][1] = contest_list[0][1], convert_sec_to_date(contest_list[0][0])
+        return contest_list
+    
+    else:
+        return None, error
+
+if __name__ == '__main__':
+    print(upcoming_contest())
```
Comment (kakshay21) [inline cf/api/contest.py]: a new line here and other places too

File: cf/api/user.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import api
+
+
+def user_profile(user_handle):
+    """ Printing user profile from Codeforces """
+    end_point_url = 'https://codeforces.com/api/user.info?handles={}'.format(user_handle)
```
Comment (kakshay21) [inline cf/api/user.py]: Once you have all things in place like every feature in the form of functions then try implementing in OOPs.
Also, read about design practices and decide one which is suitable for this.

Comment (kakshay21) [reply cf/api/user.py]: it's good for now ;)

File: cf/api/contest.py (hunk 3 above)
Comment (shubhamivane) [reply cf/api/contest.py]: Exceptions added and extra newlines are added.

**Pull request merged**